### PR TITLE
✨ LLM 📱 -  [FEAT] : LIVE-4478 : Asset Content Card

### PR DESCRIPTION
**Pull request by mcayuelas-ledger**
### 📝 Description

LLM Assets Page Content Cards

### ❓ Context

- **Impacted projects**: `live-mobile` `ui-native`
- **Linked resource(s)**: [LIVE-4478] 

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

[LIVE-4478]: https://ledgerhq.atlassian.net/browse/LIVE-4478?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/ledger-live-mobile/src/dynamicContent/dynamicContent.tsx [hunk 1]
```diff
@@ -39,6 +42,7 @@ export const filterByPage = (array: BrazeContentCard[], page: string) =>
 export const mapAsWalletContentCard = (card: BrazeContentCard) =>
   ({
     id: card.id,
+    tag: card.extras.title,
```
Comment (mcayuelas-ledger) [inline apps/ledger-live-mobile/src/dynamicContent/dynamicContent.tsx:45]: Oups wrong

File: apps/ledger-live-mobile/src/screens/WalletCentricAsset/index.tsx [hunk 2]
```diff
@@ -135,6 +143,49 @@ const AssetScreen = ({ route }: NavigationProps) => {
...
+    [],
+  );
+
+  useEffect(() => {
+    if (dynamicContentCard) {
+      // Notify Braze that the card has been displayed to the user
+      logImpressionCard(dynamicContentCard.id);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```
Comment (cgrellard-ledger) [inline apps/ledger-live-mobile/src/screens/WalletCentricAsset/index.tsx:185]: No need to put dynamicContentCard in the deps ?

File: apps/ledger-live-mobile/src/screens/WalletCentricAsset/index.tsx [hunk 3]
```diff
@@ -135,6 +143,49 @@ const AssetScreen = ({ route }: NavigationProps) => {
...
+    [logClickCard],
+  );
+
+  const onPressDismiss = useCallback(
+    (dismissAction: () => void, card: ContentCard) => {
+      track("contentcard_dismissed", {
+        screen: card.location,
+      });
+      // Braze dismiss  logDismissCard(card.id);
+      dismissAction();
```
Comment (cgrellard-ledger) [inline apps/ledger-live-mobile/src/screens/WalletCentricAsset/index.tsx:174]: We need to remove this line as we're keeping the current behavior

File: apps/ledger-live-mobile/src/screens/WalletCentricAsset/index.tsx [hunk 4]
```diff
@@ -135,6 +143,49 @@ const AssetScreen = ({ route }: NavigationProps) => {
...
+  const {
+    getAssetCardByIdOrTicker,
+    logDismissCard,
+    logClickCard,
+    logImpressionCard,
+  } = useDynamicContent();
+  const dynamicContentCard = getAssetCardByIdOrTicker(currency);
+
+  const onClickLink = useCallback(
+    (card: ContentCard) => {
```
Comment (cgrellard-ledger) [inline apps/ledger-live-mobile/src/screens/WalletCentricAsset/index.tsx:156]: Do we need to pass the content card as a parameter ? We could use `dynamicContentCard` directly instead

File: apps/ledger-live-mobile/src/screens/WalletCentricAsset/index.tsx (hunk 3 above)
Comment (mcayuelas-ledger) [reply apps/ledger-live-mobile/src/screens/WalletCentricAsset/index.tsx:174]: `dismissAction` this one is the function that you pass as param

`logDismissCard(card.id);` it's this that i commented :)

**Review by cgrellard-ledger — APPROVED** [review]

**Pull request merged**